src/platform/forms-system/test/js/components/FormSignature.unit.spec.jsx

```diff
@@ -22,23 +22,19 @@ describe('Forms library - Forms signature component', () => {
 
   describe('signature input', () => {
     it('should render input with default label', () => {
-      const { getByLabelText } = render(<FormSignature {...signatureProps} />);
-      expect(getByLabelText('Veteran’s full name')).to.exist;
+      const { container } = render(<FormSignature {...signatureProps} />);
+      expect(
+        container.querySelector('va-text-input').getAttribute('label'),
+      ).to.equal('Veteran’s full name');
     });
 
     it('should render input with custom string label', () => {
-      const { getByLabelText } = render(
+      const { container } = render(
         <FormSignature {...signatureProps} signatureLabel="Custom text here" />,
       );
-      expect(getByLabelText('Custom text here')).to.exist;
-    });
-
-    it('should render input input with custom React element label', () => {
-      const customLabel = <span>Custom text here</span>;
-      const { getByLabelText } = render(
-        <FormSignature {...signatureProps} signatureLabel={customLabel} />,
-      );
-      expect(getByLabelText('Custom text here')).to.exist;
```

Comment:
This test is no longer relevant because the web component does not allow custom HTML passed into the label like the React component did. It's also a duplicate test. The test at line 25 is already checking that it renders a provided label text.